homeassistant/components/isy994/__init__.py

```diff
@@ -192,6 +192,10 @@ async def async_setup_entry(
         raise ConfigEntryNotReady(
             f"Invalid XML response from ISY; Ensure the ISY is running the latest firmware: {err}"
         ) from err
+    except TypeError as err:
```

Comment:
Please also update the library and have it raise a library specific exception for this case.

Reply:
Will do. Already being discussed here https://github.com/automicus/PyISY/issues/297